controllers/drain_controller_test.go

```diff
@@ -90,8 +90,9 @@ var _ = Describe("Drain Controller", Ordered, func() {
 	})
 
 	BeforeEach(func() {
-		Expect(k8sClient.DeleteAllOf(context.Background(), &corev1.Node{})).ToNot(HaveOccurred())
-		Expect(k8sClient.DeleteAllOf(context.Background(), &sriovnetworkv1.SriovNetworkNodeState{}, client.InNamespace(vars.Namespace))).ToNot(HaveOccurred())
+		Expect(k8sClient.DeleteAllOf(context.Background(), &corev1.Node{}, &client.DeleteAllOfOptions{DeleteOptions: client.DeleteOptions{GracePeriodSeconds: pointer.Int64(0)}})).ToNot(HaveOccurred())
```

Comment:
we need to specify client.DeleteAllOfOptions ? or you want grace period to be 0 ?

is it related to the version bump ?

Reply:
ack ! thx for clariying